pandas/core/frame.py

```diff
@@ -7444,8 +7443,9 @@ def isin(self, values):
 
         Returns
         -------
-
-        DataFrame of booleans
+        DataFrame
+            DataFrame of boolean showing whether each element in the DataFrame
+            is contained in values.
 
```

Comment:
Can we add a `See Also` section? I think `DataFrame.eq` and `Series.str.isin` are worth referencing.